posthog/api/person.py

```diff
@@ -259,7 +288,11 @@ def destroy(self, request: request.Request, pk=None, **kwargs):  # type: ignore
             raise NotFound(detail="Person not found.")
 
     def get_queryset(self):
-        return self._filter_request(self.request, super().get_queryset())
+        queryset = super().get_queryset()
+        queryset = self.filterset_class(self.request.GET, queryset=queryset, team_id=self.team.id).qs
+        queryset = queryset.prefetch_related(Prefetch("persondistinctid_set", to_attr="distinct_ids_cache"))
```

Comment:
I don't see the attr `distinct_ids_cache` being used anywhere in this PR, is this an optimization that is referenced elsewhere?

Reply:
Yeah we use it in a couple of places, this was copied form the `filter_persons` method